store.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/store.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="css/checkbox-radio.css" >
+    <title>Karma form</title>
+</head>
+
+<body>
+    <header>
+        <nav class="navigation">
+            <!-- Hamburger menu -->
+            <div class="mobile-container">
+                <img class="logo-svg" src="img/karma-logo.svg">
+                <div class="topnav">
+
+                    <div id="myLinks">
+                        <strong><a class="nav-li" href="#">Meet Karma</a></strong>
+                        <a class="nav-li" href="#">How is work</a>
+                        <a class="nav-li" href="#">Store</a>
+                        <a class="nav-li" href="#">Blog</a>
+                        <a class="nav-li" href="#">Help</a>
+                        <a class="nav-li" href="#">Login</a>
+                    </div>
+                    <a href="javascript:void(0);" class="icon" onclick="myFunction()">
+                        <i class="fa fa-bars"></i>
+                    </a>
+                </div>
+            </div>
+            <!-- Desktop menu -->
+            <ul class="flexboxrow navul">
+                <li><img class="logo-svg" src="./img/karma-logo.svg"></li>
+                <li class="li-space"></li>
+                <li><strong><a class="nav-li" href="#">Meet Karma</a></strong></li>
+                <li><a class="nav-li" href="#">How is work</a></li>
+                <li><a class="nav-li" href="#">Store</a></li>
+                <li><a class="nav-li" href="#">Blog</a></li>
+                <li><a class="nav-li" href="#">Help</a></li>
+                <li><a class="nav-li" href="#">Login</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main class="form-image-girl-working">
+        <form style="width: 100%;">
+            <legend><h1 class="color-orange">Order your Karma wifi device today!</h1></legend>
```

Comment:
It's generally more flexible and readable to use `class` attributes sparingly, especially on small and simple pages. Instead of assigning classes directly to each element, I'd suggest to assign a class to a higher level element (like a form) and use CSS to style the nested elements. For example:

```html
        <form class="store-form">
            <legend><h1>Order your Karma wifi device today!</h1></legend>
             <fieldset> ... </fieldset>
        </form>
```

```css
.store-form h1 {
    /* ... */
}

.store-form fieldset {
    /* ... */
}
```

This helps you to think about web page blocks as higher-level `components`, not individual low-level `elements`. It's going to be very useful going forward in various situations.